src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java

```diff
@@ -69,23 +69,31 @@ public void testTokensNotNull() {
     }
 
     @Test
-    public void testWithMultilineStatements() throws Exception {
+    public void testWithMultilineStatementsOne() throws Exception {
```

Comment:
After some discussion, we cannot allow `test...One`, `test....Two` naming convention. It is better to logically split the input files so it is easier to debug and maintain it in the future.

Reply:
done